code/game/objects/items/trader.dm

```diff
@@ -1018,3 +1018,153 @@ var/global/list/alcatraz_stuff = list(
+	mech_flags = MECH_SCAN_FAIL	//Nip that in the bud
+	var/static/list/illegalChems = list(	//Just a bad idea
+		ADMINORDRAZINE,
+		BLOCKIZINE,
+		AUTISTNANITES,
+		XENOMICROBES,
+		PAISMOKE
+	)
+
+/obj/item/weapon/reagent_containers/glass/beaker/vial/mystery/New()
+	..()
+	var/list/mysteryChems = chemical_reagents_list - illegalChems
+	reagents.add_reagent(pick(mysteryChems), volume)
+
+
+//Mystery circuits////////////
+
+/obj/item/weapon/storage/box/mystery_circuit
+	name = "children's circuitry circus educational toy booster pack"
+	desc = "Ages 6 and up"
+	icon = 'icons/obj/storage/storage.dmi'
+	icon_state = "circuit"
+
+/obj/item/weapon/storage/box/mystery_circuit/New()
+	..()
+	var/list/legalCircuits = existing_typesof(/obj/item/weapon/circuitboard) - /obj/item/weapon/circuitboard/card/centcom	//Identical to spessmart spawner
+	for(var/i = 1 to 3)
+		var/boosterPack = pick(legalCircuits)
+		new boosterPack(src)
+	new /obj/item/weapon/solder(src)
+	new /obj/item/weapon/reagent_containers/glass/beaker/sulphuric(src)
+	new /obj/item/weapon/paper/permissionslip(src)
+
+/obj/item/weapon/paper/permissionslip
+	name = "circuitry circus education toy booster pack legally binding permission slip"
+	desc = "Very clearly hand written."
+
+/obj/item/weapon/paper/permissionslip/New()
+	..()
+	info = "The purchaser or purchasers of this or any other Circuitry Circus Education Toy Booster Pack <i>TM</i> recognizes, accepts, and is bound to the terms and conditions found within any Circuitry Circus Education Toy Starter Pack <i>TM</i>. This includes but is not limited to: <BR>the relinquishment of any state, country, nation, or planetary given rights protecting those of select ages from legal action based on misuse of the product.<BR>All: injuries, dismemberments, trauma (mental or physical), diseases, invasive species, deaths, memory loss, time loss, genetic recombination, or quantum displacement is the sole responsibility of the owner of the Circuitry Circus Education Toy Booster Pack <i>TM</i> <BR><BR>Please ask for your parent or guardian's permission before playing. Have fun."
+
+
+//Mystery material//////////////////////
+
+/obj/item/weapon/storage/box/large/mystery_material
+	name = "surplus material scrap box"
+	desc = "Caked in layers of dust, smells like a warehouse."
+	var/list/surplusMat= list(
+		/obj/item/stack/sheet/metal = 50,
+		/obj/item/stack/sheet/glass = 35,
+		/obj/item/stack/sheet/plasteel = 25,
+		/obj/item/stack/sheet/mineral/uranium = 20,
+		/obj/item/stack/sheet/mineral/silver = 20,
+		/obj/item/stack/sheet/mineral/gold = 15,
+		/obj/item/stack/sheet/mineral/diamond = 5,
+		/obj/item/stack/sheet/mineral/phazon = 1,
+		/obj/item/stack/sheet/mineral/clown = 1
+	)
+
+/obj/item/weapon/storage/box/large/mystery_material/odd
+	name = "surplus odd material scrap box"
+	surplusMat = list(
+		/obj/item/stack/sheet/bone = 50,
```

Comment:
sneaky coder

Reply:
Just to be clear, you really want "the chance of spawning 50 bones" to have 50 weight? I feel like you're taking a shortcut to avoid specifying both the chance and the amount

Reply:
It was definitely a shortcut but if I thought it was too limiting while making it I would have added a different way to specify amount. I can change that if you prefer.
What's the concern with 50 bones? Outside of ectoplasm they're entirely useless, and ectoplasm is more limited by the frost oil than bone marrow, however this does greatly speed up the process of making it. They were intended to be a dummy prize like metal sheets in the other one. 
I can swap cardboard and bones or whatever you'd prefer though. This box is mainly for lesser-used materials to be more attainable.

Reply:
I really don't care at all about any of this, but you did mention you were going to add some things to do with bones in the future.

Reply:
Mostly aesthetic stuff

Reply:
k